### PR TITLE
Ngrok 271

### PR DESCRIPTION
**Pull request by popkinj**
Integration of Ngrok tokens so we can pass html to the user within the dev env.
Also updated the Ngrok image to the most recent version from the officially supported source.

### CODE REVIEW

File: docker/manage [hunk 1]
```diff
@@ -330,6 +340,8 @@ function setNgrokEndpoints() {
     NGROK_AGENT_PORT=4059
   fi
 
+  sleep 60
```
Comment (esune) [inline docker/manage:343]: We should not have arbitrary sleep times in the script. What is this supposed to do?

**Review by esune — CHANGES_REQUESTED** [review]

File: docker/manage (hunk 1 above)
Comment (popkinj) [reply docker/manage:343]: It's the only thing allowing the build env to start properly.
I was thinking it's temporary. That was the new ticket I referred to in RC.

Comment (popkinj) [reply docker/manage:343]: Good eye though @esune

Comment (popkinj) [reply docker/manage:343]: If you want I can just comment it out for the PR... Then add a note beside it.

Comment (esune) [reply docker/manage:343]: I think we should figure out why this is needed, it should not be required to execute the script. DO you have any hints/ideas on why it is required? What happens when you remove it?

**Pull request merged**